### PR TITLE
Work Queue Documentation Update

### PR DESCRIPTION
**Pull request by tjdasso**
This PR includes a configuration file for an example Parsl application using the WorkQueueExecutor. It also adds the appropriate documentation to the `configuring.rst` file for WorkQueue, including basic installation using `conda-forge`.

### CODE REVIEW

File: parsl/configs/wqex_local.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from parsl.config import Config
+from parsl.executors import WorkQueueExecutor
+
+config = Config(
+    executors=[
+        WorkQueueExecutor(
+            label="wqex_local"
```
Comment (benclifford) [inline parsl/configs/wqex_local.py]: CI is failing with a flake8 error here, which I think comes from a missing comma at the end of the line.

```
$ flake8 parsl/
parsl/configs/wqex_local.py:8:16: E999 SyntaxError: invalid syntax
```

File: docs/userguide/configuring.rst [hunk 2]
```diff
@@ -113,6 +115,9 @@ In addition, examples for some specific configurations follow.
 .. [*] 8000 nodes with 32 workers each totalling 256000 workers is the maximum scale at which
        we've tested the `ExtremeScaleExecutor`.
 
+.. [*] The maximum number of nodes tested for the `WorkQueueExecutor` is 10000 GPU cores and 
+       20000 CPU cores.
```
Comment (benclifford) [inline docs/userguide/configuring.rst]: What does it mean to test on GPU cores? Did you have some specific GPU test code?

File: docs/userguide/configuring.rst [hunk 3]
```diff
@@ -338,6 +343,23 @@ configuration follows.
    `here <https://github.com/Parsl/parsl/issues/941>`_.
 
 
+WorkQueue (CCL ND)
```
Comment (benclifford) [inline docs/userguide/configuring.rst]: is it WorkQueue or Work Queue? you use both in this text

File: docs/userguide/configuring.rst [hunk 4]
```diff
@@ -338,6 +343,23 @@ configuration follows.
...
+.. image:: http://ccl.cse.nd.edu/software/workqueue/WorkQueueLogoSmall.png
+
+The following snippet shows an example configuration for using the Work Queue distributed framework to run applications on remote machines at large. This examples uses the `WorkQueueExecutor` to schedule tasks locally, and assumes that Work Queue workers have been externally connected to the master using the `work_queue_worker` or `condor_submit_workers` command line utilities from CCTools. For more information the process of submitting tasks and workers to Work Queue, please refer to the `CCTools Work Queue documentation <https://cctools.readthedocs.io/en/latest/work_queue/>`.
+
+.. literalinclude::  ../../parsl/configs/wqex_local.py
+
+To utilize Work Queue with Parsl, please install the full CCTools software package within an appropriate Anaconda or Miniconda environment (instructions for installing Miniconda can be found `here <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`):
+
+.. codeblock:: bash
+    $ conda create -y --name <environment> python=<version>
```
Comment (benclifford) [inline docs/userguide/configuring.rst]: does this environment need to be the same environment that parsl needs to be installed in?

Comment (tjdasso) [reply docs/userguide/configuring.rst]: If Parsl is installed using Pip outside of an environment, then it shouldn't be a problem. If Parsl is installed within a conda environment, then yes, the user would need to have the same environment for both Parsl and Work Queue. I'll add that to the docs

File: docs/userguide/configuring.rst (hunk 2 above)
Comment (tjdasso) [reply docs/userguide/configuring.rst]: The example Work Queue "task" for this test utilized GPU-specific code that interacted with the GPU. Work Queue handled the transfer of data and information from the user to the machine with the GPU, but yes, it was the example task program that had the GPU-specific test code.

**Pull request merged**